doc/content/getting-started/migrating-from-v2/import-devices.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Import End Devices in The Things Stack
+weight: 50
+---
+
+## Import Devices using ttn-lw-cli
+
+To import your devices, use the `devices.json` file you created
+in the previous step with `ttn-lw-cli`:
+
+```bash
+$ ttn-lw-cli dev create --application-id "imported-application" < devices.json
+```
+
+This will import your devices on {{% tts %}}. In case
+any device fails, you see a relevant error message at the end of the output.
+
+> **ΝΟΤΕ**: Keep in mind that an end device can only be registered in one Network Server at a time. After importing an end device to {{% tts %}}, you should remove it from {{% ttnv2 %}}. For OTAA devices, it is enough to simply change the AppKey, so the device can no longer connect but the existing session is preserved. Next time the device joins, it will connect to {{% tts %}}.
+
+Change the AppKey for OTAA end devices on {{% ttnv2 %}} with the Console, or by using the following command:
+
+```bash
+# set to 0000000000000000, or use any other 64-bit hex string
+$ ttnctl devices set "DeviceID" --app-key "0000000000000000"
```

Comment:
There's no "disable" counterpart like `export-all`, is there?

Do we want that?

We certainly don't want conflicting join-accepts to be sent

Reply:
We do not have that currently. It might make sense to have that, but then we would not want users to lose their AppKey's by mistake.

Reply:
Yes, that is to be avoided anyway.

How about rewriting the instructions like:

1. Export one or all devices to a file (as backup + input for 3)
2. Reset AppKey of one or all devices
3. Pipe the file to the V3 CLI end devices create command

Reply:
Moved the section for changing the AppKey after exporting.

Reply:
OK, but still it doesn't work for multiple devices at once.

It is not feasible to reset the AppKey one-by-one.

Reply:
Should this be added on ttnctl, or can we just tell the users to do something like:

```bash
devices=$(ttnctl devices list | grep -v "INFO" | grep -v "\<DevID.*AppEUI.*DevEUI.*DevAddr.*Description\>" | cut -f1)

for dev in $devices; do
  ttnctl devices update ....
done
```

Reply:
An alternative would be to add a `ttnctl devices disable` (and a `disable-all` version) that resets AppKey's of devices and keeps the old one as a device attribute (e.g. `ttnctl-backup-appkey`).

Possibly this is getting too far, but that would allow us to have a `re-enable` command, that restores the AppKey from that attribute.

Are attributes safe to use for that, or are they public? If they are, we could also print the old AppKey and tell the user to store it somewhere safe.

Reply:
Promoting the discussion, see below